### PR TITLE
feat: expose auction and secure collateral threshold getters

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: src/apis/vaults.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ export interface VaultsAPI {
     getTotalIssuedPolkaBTCAmount(): Promise<PolkaBTC>;
     selectRandomVaultIssue(btc: PolkaBTC): Promise<AccountId>;
     selectRandomVaultRedeem(btc: PolkaBTC): Promise<AccountId>;
+    getAuctionCollateralThreshold(): Promise<Big>;
```
Comment (nud3l) [inline src/apis/vaults.ts:43]: can you move them down to line 46 so they are with the other thresholds please?

File: src/apis/vaults.ts [hunk 2]
```diff
@@ -250,9 +252,14 @@ export class DefaultVaultsAPI {
         return new UInt(new TypeRegistry(), 0) as PolkaBTC;
     }
 
-    private async getSecureCollateralThreshold(): Promise<Big> {
-        const secureCollateralThresholdU128 = await this.api.query.vaultRegistry.secureCollateralThreshold();
-        return new Big(secureCollateralThresholdU128.toString());
+    async getAuctionCollateralThreshold(): Promise<Big> {
```
Comment (nud3l) [inline src/apis/vaults.ts:255]: can you also move them down to line 308 so we have it with the rest of the thresholds

also, can you transform the other thresholds to return a `Big` please?

File: src/mock/apis/vaults.ts [hunk 3]
```diff
@@ -135,6 +136,14 @@ export class MockVaultsAPI implements VaultsAPI {
         return new UInt(registry, 100) as DOT;
     }
 
+    async getAuctionCollateralThreshold(): Promise<Big> {
```
Comment (nud3l) [inline src/mock/apis/vaults.ts:139]: also, please move to the other thresholds

**Review by nud3l — APPROVED** [review]

File: src/apis/vaults.ts [hunk 4]
```diff
@@ -178,10 +180,10 @@ export class DefaultVaultsAPI {
         try {
             const collateralization = newCollateral
                 ? await customAPIRPC.vaultRegistry.getCollateralizationFromVaultAndCollateral(
-                    vaultId,
-                    this.wrapCurrency(newCollateral),
-                    onlyIssued
-                )
+                      vaultId,
```
Comment (savudani8) [inline src/apis/vaults.ts:183]: this indent might cause the lint to fail

**Pull request merged**